e2e/paths/enterChildren_test.js

```diff
@@ -68,50 +68,51 @@ Scenario('completing entering child information in the c110a application', (I, e
-  I.seeAnswerInTab(5, 'Additional children 1', 'What date did they start staying here?', '2 Nov 2017');
+  I.seeAnswerInTab(3, 'Party', 'First name', 'Susan');
+  I.seeAnswerInTab(4, 'Party', 'Last name', 'Wilson');
+  I.seeAnswerInTab(5, 'Party', 'Date of birth', '1 Jul 2016');
+  I.seeAnswerInTab(6, 'Party', 'Gender', 'Girl');
+  I.seeAnswerInTab(7, 'Party', 'Describe child\'s situation', 'Living with respondents');
+  I.seeAnswerInTab(8, 'Party', 'What date did they start staying here?', '2 Nov 2017');
+  I.seeAnswerInTab(10, 'Party', 'Key dates for this child', 'Tuesday the 11th');
   I.seeAnswerInTab(1, 'Current address', 'Building and Street', '2 Three Tuns Wynd');
   I.seeAnswerInTab(2, 'Current address', '', 'High Street');
   I.seeAnswerInTab(3, 'Current address', '', 'Stokesley');
   I.seeAnswerInTab(4, 'Current address', 'Town or City', 'Middlesbrough');
   I.seeAnswerInTab(5, 'Current address', 'Postcode/Zipcode', 'TS9 5DQ');
   I.seeAnswerInTab(6, 'Current address', 'Country', 'United Kingdom');
-  I.seeAnswerInTab(7, 'Additional children 1', 'Key dates for this child', 'Tuesday the 11th');
-  I.seeAnswerInTab(8, 'Additional children 1', 'Brief summary of care and contact plan', 'care plan summary');
-  I.seeAnswerInTab(9, 'Additional children 1', 'Are you considering adoption at this stage?', 'No');
-  I.seeAnswerInTab(10, 'Additional children 1', 'Mother\'s full name', 'Laura Smith');
-  I.seeAnswerInTab(11, 'Additional children 1', 'Father\'s full name', 'David Smith');
-  I.seeAnswerInTab(12, 'Additional children 1', 'Does the father have parental responsibility?', 'Yes');
-  I.seeAnswerInTab(13, 'Additional children 1', 'Name of social worker', 'James Jackson');
-  I.seeAnswerInTab(14, 'Additional children 1', 'Social worker\'s telephone number', '01234567');
-  I.seeAnswerInTab(15, 'Additional children 1', 'Does the child have any additional needs?', 'No');
-  I.seeAnswerInTab(16, 'Additional children 1', 'Do you need contact details hidden from other parties?', 'No');
-  I.seeAnswerInTab(17, 'Additional children 1', 'Do you believe this child will have problems with litigation capacity (understanding what\'s happening in the case)', 'No');
+  I.seeAnswerInTab(11, 'Party', 'Brief summary of care and contact plan', 'care plan summary');
+  I.seeAnswerInTab(12, 'Party', 'Are you considering adoption at this stage?', 'No');
+  I.seeAnswerInTab(13, 'Party', 'Mother\'s full name', 'Laura Smith');
+  I.seeAnswerInTab(14, 'Party', 'Father\'s full name', 'David Smith');
+  I.seeAnswerInTab(15, 'Party', 'Does the father have parental responsibility?', 'Yes');
+  I.seeAnswerInTab(16, 'Party', 'Name of social worker', 'James Jackson');
+  I.seeAnswerInTab(1, 'Social worker\'s telephone number', 'Telephone number', '01234567');
+  I.seeAnswerInTab(18, 'Party', 'Does the child have any additional needs?', 'No');
+  I.seeAnswerInTab(19, 'Party', 'Do you need contact details hidden from other parties?', 'No');
 });
```

Comment:
are you missing the assertion for "Do you believe this child will have problems with litigation capacity (understanding what\'s happening in the case)"

Reply:
Right you are, added